src/test/java/net/glowstone/entity/GlowPlayerTest.java

```diff
@@ -57,7 +59,7 @@
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
-@PrepareForTest({Bukkit.class, ChunkManager.class})
+@PrepareForTest( {Bukkit.class, ChunkManager.class})
```

Comment:
Delete the space before the curly bracket here.

Reply:
Done.